src/plone/restapi/tests/http-examples/controlpanels_get_rule.resp

```diff
@@ -350,7 +350,6 @@ Content-Type: application/json
                 "required": [
                     "subject",
                     "recipients",
-                    "exclude_actor",
```

Comment:
Just out of curiosity. Does anyone know why "exclude_actor" was removed in Plone 6.0.4? cc @mauritsvanrees

Reply:
This is for content rules, right?  No idea, I never use them. 
This is the diff in versions:

```
6.0.3.cfg:plone.app.contentrules = 5.0.0
6.0.4.cfg:plone.app.contentrules = 5.0.2
6.0.3.cfg:plone.contentrules = 2.1.3
6.0.4.cfg:plone.contentrules = 3.0.0
```

Seems to be just updating config files, dropping Py 2 support, stuff like that.